hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisor.java

```diff
@@ -240,6 +242,7 @@ public void addTask(AbstractReplicationTask task) {
           failureCounter.put(task.getMetricName(), new AtomicLong(0));
           timeoutCounter.put(task.getMetricName(), new AtomicLong(0));
           skippedCounter.put(task.getMetricName(), new AtomicLong(0));
+          requestTotalTime.put(task.getMetricName(), new AtomicLong(0));
```

Comment:
Thanks @errose28  for the comment and review.
I will update it later.
Right now, taskCounter records the number of queues.
```
private final Map<Class<?>, AtomicInteger> taskCounter =
      new ConcurrentHashMap<>();
```
I will add the QueueCount metric to ReplicationSupervisorMetrics later.
In addition, I suggest improving taskCounter so that it is consistent with requestCounter and successCounter.
```
private final Map<String, AtomicLong> taskCounter = new ConcurrentHashMap<>();
```
What do you think?